fail2ban/client/jailsreader.py

```diff
@@ -50,6 +50,7 @@ def jails(self):
 		return self.__jails
 
 	def read(self):
+		self.__jails = list()
```

Comment:
yep, this is interactive only :(

see #825, example:
fail2ban-client start
fail2ban-client -i
... and make reload twice ...